src/histolab/slide.py

```diff
@@ -64,6 +90,37 @@ def __repr__(self):
 
     # ---public interface methods and properties---
 
+    @lazyproperty
+    def base_mpp(self) -> float:
+        """Get microns-per-pixel resolution at scan magnification.
+
+        Returns
+        -------
+        float
+            Microns-per-pixel resolution at scan (base) magnification.
+
+        """
+        if self._use_largeimage:
+            return self._metadata["mm_x"] * (10 ** 3)
```

Comment:
I think we have an edge case here but maybe I'm wrong. If you add this case `(SVS.TCGA_CR_7395_01A_01_TS1, (128, 128), 0, 2, 10, 0.5)` in `test_extract_tiles_respecting_the_given_tile_size` it will throw a `ValueError` because `self._metadata["mm_x"]` is None ... here the proof:
```
{'levels': 6, 'sizeX': 4443, 'sizeY': 3567, 'tileWidth': 256, 'tileHeight': 256, 'magnification': None, 'mm_x': None, 'mm_y': None}
```

I think we should handle this case. What do you think?

Reply:
ok, handled.